plugins/woocommerce/tests/e2e-pw/tests/merchant/products/block-editor/disable-block-product-editor.spec.js

```diff
@@ -0,0 +1,86 @@
+const { test, expect } = require( '@playwright/test' );
+const {
+	clickAddNewMenuItem,
+	expectBlockProductEditor,
+	expectOldProductEditor,
+	isBlockProductEditorEnabled,
+	toggleBlockProductEditor,
+} = require( '../../../../utils/simple-products' );
+
+const ALL_PRODUCTS_URL = 'wp-admin/edit.php?post_type=product';
+const NEW_EDITOR_ADD_PRODUCT_URL =
+	'wp-admin//admin.php?page=wc-admin&path=%2Fadd-product';
```

Comment:
Probably not going to cause an issue, but you have an extra `/` in the URL here.